### PR TITLE
[Unblocked] Add domains and simplify ruleset

### PR DESCRIPTION
**Pull request by Bisaloo**
#10843

### CODE REVIEW

File: src/chrome/content/rules/Unblocked.xml [hunk 1]
```diff
@@ -1,33 +1,73 @@
 <!--
-	HSTS: max-age=15552000; preload || Not in the list yet
+	Other domains:
+		unblocked.pub
```
Comment (J0WI) [inline src/chrome/content/rules/Unblocked.xml]: WFM

File: src/chrome/content/rules/Unblocked.xml [hunk 2]
```diff
@@ -1,33 +1,73 @@
...
+		unblckd.biz
```
Comment (J0WI) [inline src/chrome/content/rules/Unblocked.xml]: Cannot resolve

**Comment by J0WI** [comment]
I think it's safe to keep a wildcard here, because every mirror get his own subdomain.

File: src/chrome/content/rules/Unblocked.xml (hunk 1 above)
Comment (Bisaloo) [inline src/chrome/content/rules/Unblocked.xml]: Yeah, I was too lazy to add it but I will do it.

File: src/chrome/content/rules/Unblocked.xml (hunk 2 above)
Comment (Bisaloo) [inline src/chrome/content/rules/Unblocked.xml]: WFM

**Comment by Bisaloo** [comment]
> I think it's safe to keep a wildcard here, because every mirror get his own subdomain.

I didn't see this comment. I came here to ask you this. Glad to see we're on the same page.

**Pull request merged**